### PR TITLE
Return fees from Depositor.withdraw + misc

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: src/funnels/Depositor.sol [hunk 1]
```diff
@@ -74,16 +74,16 @@ contract Depositor {
...
 
     event Rely(address indexed usr);
     event Deny(address indexed usr);
     event SetLimits(address indexed gem0, address indexed gem1, uint64 hop, uint128 cap0, uint128 cap1);
-    event Deposit(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1);
-    event Withdraw(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1, uint256 collected0, uint256 collected1);
-    event Collect(address indexed sender, address indexed gem0, address indexed gem1, uint256 collected0, uint256 collected1);
+    event Deposit (address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1);
+    event Withdraw(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1, uint256 fees0, uint256 fees1);
+    event Collect (address indexed sender, address indexed gem0, address indexed gem1, uint256 fees0, uint256 fees1);
```
Comment (sunbreak1211) [inline src/funnels/Depositor.sol]: I think we should remove the extra spaces excepting we really want to start aligning all the events.

**Review by oldchili — APPROVED** [review]
Apart from the removal of spaces in the Depositor events declarations this LGTM.

**Review by sunbreak1211 — APPROVED** [review]
Same, excepting that minor comment looks good to me.

File: src/funnels/Depositor.sol (hunk 1 above)
Comment (sunbreak1211) [inline src/funnels/Depositor.sol]:
```suggestion
    event Deposit(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1);
    event Withdraw(address indexed sender, address indexed gem0, address indexed gem1, uint128 liquidity, uint256 amt0, uint256 amt1, uint256 fees0, uint256 fees1);
    event Collect(address indexed sender, address indexed gem0, address indexed gem1, uint256 fees0, uint256 fees1);
```

**Pull request merged**